### PR TITLE
Ncit/ncba changes

### PR DESCRIPTION
**Pull request by ry-oc**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/national-qualifications-framework/ncba/assessments/ncba-sa4-design-thinking-empathise-feedback-data/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Please do the following:
+
+1. Follow [this link](https://docs.google.com/document/d/1NI2g3uT4Pk19iB2b82kSwef6003sB_mP/edit?usp=sharing&ouid=106698657596806218419&rtpof=true&sd=true) and download the document. If you have an Umuzi email address then you will have access to the document. If you do not have an Umuzi email address you may need to request access.
+
+2. Complete the assessment. The instructions are in the document. 
+   
+3. Upload the document to google drive and submit a link. Ensure that the submission has the `docx` file extension. Please make sure that the link works. If your work is private then we won't be able to review it. You can double-check this by opening your own link in an incognito browser.  
+
+4. Your link should look like this:
```
Comment (sheenarbw) [inline content/national-qualifications-framework/ncba/assessments/ncba-sa4-design-thinking-empathise-feedback-data/_index.md]: Maybe say "your link would look something like this:"

It's good to be explicit about the fact that this is not exact.

File: content/national-qualifications-framework/ncit/content/database-development/_index.md [hunk 2]
```diff
@@ -82,8 +82,7 @@ ncit_standards:
 - 114048
 - 114049
 prerequisites:
-  hard:
-  - sql/shop-project
```
Comment (sheenarbw) [inline content/national-qualifications-framework/ncit/content/database-development/_index.md:86]: are you sure? I know this is a prerequisite elsewhere but the thing about graphs is that they can have multiple links between nodes. It doesn't have to be a tree, it's more like a roadmap.

File: content/national-qualifications-framework/ncit/content/test-driven-development/_index.md [hunk 3]
```diff
@@ -49,8 +49,7 @@ ncit_standards:
 - 115359
 - 115384
 prerequisites:
-  hard:
-  - language-agnostic/validate-id-number
```
Comment (sheenarbw) [inline content/national-qualifications-framework/ncit/content/test-driven-development/_index.md:53]: are you sure?

**Review by sheenarbw — APPROVED** [review]
ok, it looks alright. Feel free to merge if you are sure. I left some comments on stuff I am not sure about. And I think one thing can be made more explicit

File: content/national-qualifications-framework/ncit/content/test-driven-development/_index.md (hunk 3 above)
Comment (ry-oc) [reply content/national-qualifications-framework/ncit/content/test-driven-development/_index.md:53]: Yeah, I'm sure, basically moving these as they project should not block the topic. but the project should block the assessment

**Pull request merged**